### DISCUSSION TITLE
Multiprocessing compatibility

### DISCUSSION THREAD

**Discussion by vihantyagi** [plasma-umass/scalene#401] [Q&A]
I have created a sample python script to test out multiprocessing compatibility with scalene. I'm not getting any profiling data for the multiprocessed function (hello).  This is the command I used: scalene --html --outfile profile.html sample_2.py
I have also tried using decorators and scalene_profiler.start() and scalene_profiler.stop().
Please refer the ss below and let me know what I'm doing wrong. 

![scalene](https://user-images.githubusercontent.com/47202142/172535411-be5c7b90-54e4-44da-87ae-84101a7a2b6b.jpg)

Thank you for your help.

**Comment by emeryberger** [comment]
Please attach the file and make any needed mods so that we can test with it - thanks in advance!
